src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlCommand/SqlCommandCompletedTest.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Data;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests
+{
+    public static class SqlCommandCompletedTest
+    {
+        private static readonly string s_connStr = (new SqlConnectionStringBuilder(DataTestUtility.TcpConnStr) { PacketSize = 512 }).ConnectionString;
+        private static bool completedHandlerExecuted = false;
+
+        [CheckConnStrSetupFact]
```

Comment:
Should use new style:
`[ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]`